src/css/styles.css

```diff
@@ -104,24 +133,25 @@ h4 {
   color: white;
 }
 
+table {
+  margin-left: 12px;
+  padding-right: 20px;
+}
+
 .main-sections {
   display: flex;
   flex: 1;
-  color: black;
   overflow: hidden;
 }
 
 .activity {
   width: 48%;
-  background-color: #fd7326;
+  /* background-color: #fd7326; */
+  background-color: #f3621d;
```

Comment:
Will run this new background color through lighthouse to see if the contrast is good.